.gitignore

```diff
@@ -41,4 +41,7 @@ override.tf.json
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
 
-.env
\ No newline at end of file
+.env
+
+#DS local files
+.DS_Store/*
```

Comment:
This is related to the local machine, it is better to add it to the global ignore than adding them in all project. 

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
i think both, if the contributor doesn't have this in their local we will still get them :D